handbook/marketing/content.md

```diff
@@ -4,13 +4,51 @@ At Sourcegraph, we define **content marketing** as generating awareness, buildin
 
 **Content strategy** dives a layer deeper to include the governance and operations of content: how we generate, validate, and approve ideas; how we ensure quality and consistency; where we distribute content; how we measure performance; and how we update, revise, and retire content.
 
+[Read the FY22 Content Strategy.](https://docs.google.com/document/d/1uz9AN5ZiGkbVD5mQmoKNO4mdbKaI1lukQcxPB2JiPx4/edit#)
+
+## Table of contents 
+
+1. [Quick links]
+1. [Editorial process]
```

Comment:
```suggestion
```
No need to add Table of contents since it will generate automatically for all of the primary headings.